libkbfs/kbfs_ops.go

```diff
@@ -754,6 +755,36 @@ func (fs *KBFSOpsStandard) GetNodeMetadata(ctx context.Context, node Node) (
 	return ops.GetNodeMetadata(ctx, node)
 }
 
+// TeamNameChanged implements the KBFSOps interface for KBFSOpsStandard
+func (fs *KBFSOpsStandard) TeamNameChanged(
+	ctx context.Context, tid keybase1.TeamID) {
+	fs.log.CDebugf(ctx, "Got TeamNameChanged for %s", tid)
+	fs.opsLock.Lock()
+	defer fs.opsLock.Unlock()
+	// We have to search for the tid since we don't know the old name
+	// of the team here.  Should we add an index for this?
+	for fb, fbo := range fs.ops {
+		if fb.Tlf.Type() != tlf.SingleTeam {
+			continue
```

Comment:
Oh that makes sense. Somehow I was (mistakenly) thinking this was iterating through a set of notifications for different teams.